tutorials/energy-join/01-energy-join-weather-in-pandas.ipynb

```
@@ -1107,89 +1097,138 @@
       "ActivityStarted, get_enricher\n",
-      "ActivityCompleted: Activity=get_enricher, HowEnded=Success, Duration=3.8 [ms]\n",
-      "ActivityStarted, enrich_customer_data_with_agg\n",
+      "ActivityCompleted: Activity=get_enricher, HowEnded=Success, Duration=19.39 [ms]\n",
+      "ActivityStarted, enrich_customer_data_no_agg\n",
       "ActivityStarted, enrich\n",
       "Target paths: ['/year=2017/month=8/']\n",
       "Looking for parquet files...\n",
       "Reading them into Pandas dataframe...\n",
       "Reading ISDWeather/year=2017/month=8/part-00003-tid-1321158002197267978-8e3eb092-4b7a-42de-97ee-e23297ed8955-121.c000.snappy.parquet under container isdweatherdatacontainer\n",
       "Done.\n",
-      "ActivityCompleted: Activity=enrich, HowEnded=Success, Duration=32166.32 [ms]\n",
-      "ActivityCompleted: Activity=enrich_customer_data_with_agg, HowEnded=Success, Duration=32199.33 [ms]\n",
-      "[2019-04-26 07:40:00.921094] End enriching...\n"
+      "ActivityCompleted: Activity=enrich, HowEnded=Success, Duration=39574.97 [ms]\n",
+      "ActivityCompleted: Activity=enrich_customer_data_no_agg, HowEnded=Success, Duration=39606.91 [ms]\n",
+      "[2019-04-29 10:13:28.810135] End enriching...\n"
      ]
     }
    ],
    "source": [
     "import os.path\n",
     "import pandas as pd\n",
+    "import numpy as np\n",
     "from azureml.contrib.opendatasets.accessories.location_data import LatLongColumn\n",
     "from azureml.contrib.opendatasets.accessories.location_time_customer_data \\\n",
     "    import LocationTimeCustomerData\n",
     "from azureml.contrib.opendatasets import NoaaIsdWeather\n",
+    "from azureml.contrib.opendatasets.environ import PandasEnv\n",
     "\n",
     "\n",
     "if os.path.exists('./nyc_energy_enriched.csv'):\n",
-    "    print('nyc_energy_enriched.csv exists already.')\n",
+    "    raise RuntimeError('nyc_energy_enriched.csv exists already.')\n",
```

Why raise error instead of print here?